src/systems/a320_systems_wasm/src/lib.rs

```diff
@@ -72,10 +72,14 @@ async fn systems(mut gauge: msfs::Gauge) -> Result<(), Box<dyn Error>> {
             .provides_aircraft_variable("AIRSPEED INDICATED", "Knots", 0)?
             .provides_aircraft_variable("AIRSPEED MACH", "Mach", 0)?
             .provides_aircraft_variable("AIRSPEED TRUE", "Knots", 0)?
+            .provides_aircraft_variable("AMBIENT DENSITY", "Slugs per cubic feet", 0)?
             .provides_aircraft_variable("AMBIENT PRESSURE", "inHg", 0)?
             .provides_aircraft_variable("AMBIENT TEMPERATURE", "celsius", 0)?
             .provides_aircraft_variable("AMBIENT WIND DIRECTION", "Degrees", 0)?
             .provides_aircraft_variable("AMBIENT WIND VELOCITY", "Knots", 0)?
```

Comment:
Do we still need these? If we do and they have issues, can we just switch to your new method?

Reply:
Not sure those have issues I did not test. Only used in ADIRS right now, I'd say we can keep them as reading those is simple enough, so we avoid recomputing those from our wind 3D vector.  I'll have a check when possible to make sure those 2 are indeed really valid ingame